### DISCUSSION TITLE
Why spotlightClicks has no click event

### DISCUSSION THREAD

**Discussion by chinyifei** [gilbarbara/react-joyride#874] [Q&A] [answered]
Why is spotlightClicks a boolean rather than a callback function,I need to click its callback  to do something.

**Comment by gilbarbara** [comment, marked as answer]
Hey @bilibilinatural 

It allows you to click something thru the spotlight, so you can add a listener to your element.

**Reply by chinyifei** [reply]
Thanks for your reply, Can I customize the highlighted element click event at each steps,for example, click the highlighted element to execute the next step. If so, I'm sorry I didn't find the relevant document introduction or demo case. Can I get a demo?

**Reply by gilbarbara** [reply]
Demos: https://react-joyride.com/
Source code: https://github.com/gilbarbara/react-joyride-demo

**Reply by gilbarbara** [reply]
If you want to control the tour externally, check the Controlled demo

**Reply by chinyifei** [reply]
Thanks very much.
